chapters/publication.tex

```diff
@@ -1,96 +1,171 @@
+This means that updates will never be accidentally excluded,
+and it further means that updating results will not become more difficult
+as the number of inputs grows,
+because they are all managed by a single integrated process.
+
+You will note that this is not possible in tools like Microsoft Office.
+In Word, for example, you have to copy and paste each object individually
+whenever tables, graphs or other inputs have to be updated.
+This means that both the features above are not available:
+fully updating the document becomes more and more time-consuming
+as the number of inputs increases,
+and it therefore becomes more and more likely
+that a mistake will be made or something will be missed.
+Furthermore, it is very hard to simultaneously edit or track changes
+in a Microsoft Word document.
+It is usually the case that a file needs to be passed back and forth
+and the order of contributions strictly controlled
+so that time-consuming resolutions of differences can be avoided.
+Therefore this is a broadly unsuitable way to prepare technical documents.
+
+There are a number of tools that can be used for dynamic documents.
+They fall into two broad groups --
+the first compiles a document as part of code execution,
+and the second operates a separate document compiler.
+In the first group are tools such as R's RMarkdown and Stata's \texttt{dyndoc}.
+These tools ``knit'' or ``weave'' text and code together,
+and are programmed to insert code outputs in pre-specified locations.
+Documents called ``notebooks'' (such as Jupyter) work similarly,
+as they also use the underlying analytical software to create the document.
+These types of dynamic documents are usually appropriate for short or informal materials
+because they tend to offer limited editability outside the base software
+and often have limited abilities to incorporate precision formatting.
+
+On the other hand, some dynamic document tools do not require
+operation of any underlying software, but simply require
+that the writer have access to the updated outputs.
+One very simple one is Dropbox Paper, a free online writing tool
+that allows linkages to files in Dropbox,
+which are then automatically updated anytime the file is replaced.
+Like the first class of tools, Dropbox Paper has very limited formatting options,
+but it is appropriate for working with collaborators who are not using statistical software.
+However, the most widely utilized software
+for dynamically managing both text and results is \LaTeX\ (pronounced ``lah-tek'').\sidenote{
+  \url{https://www.maths.tcd.ie/~dwilkins/LaTeXPrimer/GSWLaTeX.pdf}}
+  \index{\LaTeX}
+While this tool has a significant learning curve,
+its enormous flexibility in terms of operation, collaboration,
+and output formatting and styling
+makes it the primary choice for most large technical outputs today.
+In fact, \LaTeX\ operates behind-the-scenes in many of the tools listed in the first group.
+
+\subsection{Technical writing with \LaTeX}
```

Comment:
I think the section on LaTeX should come before the section with tools such as DropBox paper, Rmarkdown and dyndoc, as LaTeX has proven to not be a fad and has been the staple of this for decades.

Reply:
It's currently after because it is a special case of dynamic document. I don't see how it could be reversed, but I clarified a bit in a68c08b545843d1b958c92760a84fd6b011ac087. Improvements here welcome.

Reply:
In terms of structure, i think it would be useful to tie these tools into a workflow. Something like - dynamic docs are helpful for exploratory analysis / preparing intermediate outputs, but latex is best practice for the actual manuscript.

Reply:
moved to #348